src/execution_plan/execution_plan.c

```diff
@@ -515,7 +515,7 @@ ExecutionPlan* _NewExecutionPlan(RedisModuleCtx *ctx, AST *ast, ResultSet *resul
         aggregate = ReturnClause_ContainsAggregation(ast->returnNode);
     }
 
-    if(ast->returnNode || ast->withNode) {
+    if(ast->returnNode || ast->withNode){
```

Comment:
Keep the space: `if(ast->returnNode || ast->withNode) {`

Reply:
done